cmd/root.go

```diff
@@ -48,8 +48,12 @@ func newRootCommand(gs *state.GlobalState) *rootCommand {
 		SilenceUsage:      true,
 		SilenceErrors:     true,
 		PersistentPreRunE: c.persistentPreRunE,
+		Version:           versionString(),
 	}
 
+	rootCmd.SetVersionTemplate(
+		`{{with .Name}}{{printf "%s " .}}{{end}}{{printf "v%s" .Version}}\n`,
```

Comment:
```suggestion
		`{{with .Name}}{{printf "%s " .}}{{end}}{{printf "v%s" .Version}}`,
```
Is it required? On Alacritty I get the following output:
```
k6 v0.47.0 (go1.21.1, linux/amd64)\n% 
```

In any case, even after removing the newline the `%` char still remains. It requires a bit of investigation about why.

Reply:
My fault on this. It was the wrong place to include the newline character.